website/docs/docs/contributing/building-a-new-adapter.md

```diff
@@ -14,16 +14,15 @@ dbt "adapters" are responsible for _adapting_ dbt's functionality to a given dat
 This guide will walk you through the first two steps, and provide some resources to help you validate that your new adapter is working correctly.
 
 ## Scaffolding a new adapter
-
-dbt comes equipped with a script which will automate a lot of the legwork in building a new adapter. This script will generate a standard folder structure, set up the various import dependencies and references, and create namespace packages so the plugin can interact with dbt. You can find this script in the dbt repo in dbt's [scripts/](https://github.com/dbt-labs/dbt-core/blob/HEAD/core/scripts/create_adapter_plugins.py) directory.
+To create a new adapter plugin you will use the [dbt-database-adapter-scaffold](https://github.com/dbt-labs/dbt-database-adapter-scaffold) to trigger an interactive session which will allow to replace default naming, author information, versioning, and be used to augment other variables in the generation of your starting project.
```

Comment:
I noticed this line above:

> you'll probably need to build a new adapter, or extend an existing one

I think it would be worth mentioning here that, if your database is forked off / wire-compatible with another database (e.g. Postgres or Apache Spark), you might be able to skip a lot of the scaffolding / setup work by just creating a thin wrapper around the already-existing plugin.

E.g. the amount of code in the [`dbt-materialize` plugin](https://github.com/MaterializeInc/materialize/tree/main/misc/dbt-materialize/dbt) is really very small. If they were to create it today, I don't think it would be worth them going through the scaffold—most expedient would just be subclassing `dbt-postgres` right from the outset.

@dataders Curious to hear your thoughts here too